chsdi/static/js/ga-debug.js

```diff
@@ -57721,9 +57721,9 @@ ga.layer.create = function(layer, options) {
         return layerConfig['label'];
       }
     },
-    queryable: {
```

Comment:
The changes in this file should not be in this PR. We should update the ol3 project first and re-create the files from there. Please remove the changes from this PR.